Vonk.Plugin.DocumentOperation.Test/VonkTestContext.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Hl7.Fhir.Model;
```

Comment:
I think this file is coming from Vonk.Test.Utils. Maybe we should also publish this package to the public, so that everybody can use it. But fine for now..

Reply:
Created ticket for this on the backlog.